trillian/docs/ManualDeployment.md

```diff
@@ -153,6 +153,9 @@ deployment process.
    future deployment scenarios where the Trillian services are operated by a
    different entity than the CT personality, and so a trust boundary between
    the two is needed.)
+ - The `--max_root_duration` option should be set to less than the log's MMD.
```

Comment:
Should there be any mention of a safety margin?

Reply:
Done.